data.sql

```diff
@@ -4,3 +4,19 @@ VALUES
   ('Gabumon', '2018-11-15', 2, TRUE, 8),
   ('Pikachu', '2021-01-07', 1, FALSE, 15.04),
   ('Devimon', '2017-05-12', 5, TRUE, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  ('Charmander', '2020-02-08', 0, FALSE, -11),
+  ('Plantmon', '2021-11-15', 2, TRUE, -5.7),
+  ('Squirtle', '1993-04-02', 3, FALSE, -12.13),
+  ('Angemon', '2005-06-12', 1, TRUE, -45),
+  ('Boarmon', '2005-06-07', 7, TRUE, 20.4),
+  ('Blossom', '1998-10-13', 3, TRUE, 17),
+  ('Ditto', '2022-05-14', 4, TRUE, 22);
+
+BEGIN;
+ALTER TABLE animals
+RENAME COLUMN species TO unspecified;
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
- [ ] After `ROLLBACK` you should verify the change was made by adding another `SELECT * FROM animals;`


## _Please note that this whole transaaction has to be moved to `queries.sql`_

Reply:
Made changes
![Screenshot 4](https://github.com/torobucci/vet_clinic/assets/103370560/ec2c80d2-c6fb-4c43-8c10-0cf9ea5cf23e)